tests/cases/conformance/types/tuple/named/partiallyNamedTuples.ts

```diff
@@ -21,24 +21,8 @@ type ToMixedTuple<T extends unknown[]> = {
   [K in keyof T]: [K, second: T[K], keyof T, fourth: T];
 };
 
-type AnonymousToMixed = ToAnonymousTuple<[boolean, number]>;
-type MixedToMixed = ToAnonymousTuple<[boolean, second: number]>;
-type NamedToMixed = ToAnonymousTuple<[first: boolean, second: number]>;
+type AnonymousToMixed = ToMixedTuple<[boolean, number]>;
+type MixedToMixed = ToMixedTuple<[boolean, second: number]>;
+type NamedToMixed = ToMixedTuple<[first: boolean, second: number]>;
 
 type MixedSpread = [first: boolean, ...[second: string]];
-
-type ConditionalTuple = [
-  first: boolean,
-  ...(0 extends 0 ? [second: string] : [])
-];
-
-type AddMixedConditional<T> = [
-  first: boolean,
-  null,
-  third: T extends number ? "a" : "b",
-  ...(T extends 0 ? [fourth: "c"] : [])
-];
-
-type AddMixedConditionalBoolean = AddMixedConditional<boolean>;
-type AddMixedConditionalLiteral = AddMixedConditional<0>;
-type AddMixedConditionalNumberPrimitive = AddMixedConditional<number>;
```

Comment:
those were pretty good tests though! :)

Reply:
Haha, thanks, adding them back. I think I've lost you on the suggested changes to tests - are you proposing I change them still?

Reply:
Sorry if I wasn't clear enough. I just think that it's interesting to verify that an input tuple with mixed members is "transformed" correctly and that we get mixed members in the output tuple. Currently, both mapped types in the tests  (`ToAnonymousTuple` and `ToMixedTuple`) check that but they also try to exercise different tuples (anonymous and mixed) in their "templates". And I just think that this part is just a little bit redundant - both could map each element to `1` or some other dummy value. To rephrase: I wanted to ensure that mixed labels from the input tuple are the same as the ones in the output tuple and the actual element types of the output tuple are not that relevant (just their labels)

Reply:
it might also be worth checking if we can infer reverse-mapped types with mixed labels from tuples with mixed labels, something like this:
```ts
declare function test<T extends readonly unknown[]>(
  arg: [
    ...{
      [K in keyof T]: {
        type: T[K];
      };
    }
  ]
): T;

declare const input: [first: { type: number }, { type: string }];

const output = test(input); // should be [first: number, string]

```